EXILED/Exiled.CustomItems/Events/MapHandler.cs

```diff
@@ -15,13 +16,22 @@ namespace Exiled.CustomItems.Events
     /// </summary>
     internal sealed class MapHandler
     {
-        /// <inheritdoc cref="Exiled.Events.Handlers.Map.Generated"/>
-        public void OnMapGenerated()
+        /// <inheritdoc cref="Exiled.Events.Handlers.Server.WaitingForPlayers"/>
+        public void OnWaitingForPlayers()
         {
-            Timing.CallDelayed(0.5f, () => // Delay its necessary for the spawnpoints of lockers and rooms to be generated.
+            Timing.CallDelayed(2, () => // The delay is necessary because the generation of the lockers takes time, due to the way they are made in the base game.
```

Comment:
This would be good if exiled OnWaitingPlayer get call when locker have finish to spawn too

Reply:
![imagen](https://github.com/user-attachments/assets/33680f0a-9286-43d1-ba69-383c842dffa4)
And how do you know when the lockers are created?


The Exiled lockers are created in the Awake of the base game lockers but the only way to know if they are already created would be to increase the delay that OnWaitingForPlayers already has.

Reply:
Fuck that kinda problematic yeah